tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/PackageVersionsLatestMajors.g.cs

```diff
@@ -26,26 +26,23 @@ public class PackageVersionsLatestMajors
 #if DEFAULT_SAMPLES
                 new object[] { string.Empty },
 #else
-#if NET462
```

Comment:
Ok definitvely weird. Maybe that's linked to the fact that you don't have .NET Fx on your mac? 🤷  It's been a while since someone with only a mac has done this

Reply:
.NET 462 is, I think, unsupported/unavailable on mac
Do I have to re-add these lignes manually?